README.md

```diff
@@ -1 +1,124 @@
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| wangyifei | <wangyifei@kubesphere.io> |  |
+
+## Source Code
+
+* <https://github.com/OpenFunction/OpenFunction>
+
+## Requirements
+
+Kubernetes: `>=v1.20.0-0`
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://dapr.github.io/helm-charts/ | dapr | 1.5.1   |
+| https://kedacore.github.io/charts | keda | 2.4.0   |
+| https://kubernetes.github.io/ingress-nginx | ingress-nginx | 4.0.6   |
+| https://openfunction.github.io/charts/ | knative-serving | 0.6.0   |
+| https://openfunction.github.io/charts/ | shipwright-build | 0.6.0   |
+| https://openfunction.github.io/charts/ | tekton-pipelines | 0.6.0   |
+
+## Install the Chart
+
+Ensure Helm is initialized in your Kubernetes cluster.
+
+For more details on initializing Helm, [read the Helm docs](https://helm.sh/docs/)
+
+1. Add `openfunction.github.io` as an helm repo
+    ```
+    helm repo add openfunction https://openfunction.github.io/charts/
+    helm repo update
+    ```
+
+2. Install the openfunction chart on your cluster in the `openfunction` namespace:
+   * If your environment does not contain any openfunction-dependent components and you want to install all components 
+   directly, You can install openfunction with all dependencies with the following command:
+      ```
+      kubectl create namespace openfunction
+      helm install openfunction openfunction/openfunction -n openfunction
+      ```
+   * If your environment already contains some of the openfunction-dependent components, or if you want to install some
+   components separately for more flexibility and customizable capabilities. For example, if you already have dapr 
+   installed in your environment, You can install openfunction with the following command:
+      ```
+      kubectl create namespace openfunction
+      helm install openfunction --set Dapr.enabled=false openfunction/openfunction -n openfunction
+      ```
+   * If you have limited network access to `gcr.io`, such as Chinese users, you can specify the version of openfunction 
+   to install, For example, if the common version is 0.1.0, the corresponding version of region-cn is 0.1.0-rc.0:
+      ```
+      kubectl create namespace openfunction
+      helm install openfunction --version 0.1.0-rc.0 openfunction/openfunction -n openfunction
+      ```
+
+## Verify installation
+
+```
+kubectl get pods -namespace openfunction
+```
+
+## Uninstall the Chart
+
+To uninstall/delete the `openfunction` release:
+1. You may need to remove the `ingresses.networking.internal.knative.dev` resource 
+from the namespace `knative-serving` first, otherwise it may block the uninstallation of knative-serving, you can 
+confirm it with the following command:
+    ```
+    kubectl get ingresses.networking.internal.knative.dev -n knative-serving
+    ```
+2. Uninstall the helm release:
+    ```
+    helm uninstall openfunction -n openfunction
```

Comment:
Will this uninstall remove openfunction itself together with all it's dependencies?

Reply:
yeah

Reply:
That might not be ok because these components might be installed separately by users for other use cases.
We should only delete openfunction itself with this uninstall cmd.
and we can add another flag like `--remove-all-dependencies` to do this explicitly

Reply:
We will only remove components installed via `helm install`, separately managed components will not be affected